### PR TITLE
no mosaic for cancer

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR disables Mosaic Tags for cancer variants

**How to test**:
1. Install on stage

**Expected outcome**:
See that the Mosaic Tags form is gone for cancer, but not for SNVs and INDELS

**Review:**
- [x] code approved by dNil
- [x] tests executed by dNil

### CODE REVIEW

**Comment by dnil** [comment]
Nice! If you name your next commit that updates the CHANGELOG something including "Fix #1243" the issue will close automagically when the PR is merged.

**Comment by mayabrandi** [comment]
Thanks for the reminder :)

> 26 nov. 2019 kl. 11:29 skrev Daniel Nilsson <notifications@github.com>:
> 
> Nice! If you name your next commit that updates the CHANGELOG something including "Fix #1243 <https://github.com/Clinical-Genomics/scout/issues/1243>" the issue will close automagically when the PR is merged.
> 
>

**Comment by northwestwitch** [comment]
Is this ready to be merged?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
+
+## [x.x.x]
+
+### Fixed
+- Remooved Mosaic Tag from Cancer variants
```
Comment (northwestwitch) [inline CHANGELOG.md]:
```suggestion
- Removed Mosaic Tag from Cancer variants
```

**Comment by northwestwitch** [comment]
Merging this in 3,2,1..

**Pull request merged**